R/buildQset.R

```diff
@@ -122,7 +146,7 @@ getBamCoveragePairedAndUnpairedR1 <- function(fileName = NULL, BSgenome = NULL,
 
   numPairsWithinSize <- length(properPairsGRanges)
 
-  if (numPairsWithinSize < numPairsInit) {
+  if (numPairsWithinSize < numPairsAfterMAPQ) {
```

Comment:
This was a bug, meaning it was reporting the wrong number for the message below.